app/views/admin/schools/index.html.erb

```diff
@@ -1,13 +1,6 @@
-<% content_for :before_content do %>
```

Comment:
What was the reason to remove this?

Reply:
I've moved this interface to use the header bar tabs so there's no need for a back button when this is a top level page within the nav bar that's always present.